src/agent/rustjail/src/mount.rs

```diff
@@ -265,6 +265,16 @@ pub fn init_rootfs(
         }
     }
 
+    
+    // Mount /tmp as tmpfs with MS_NODEV and MS_STRICTATIME
```

Comment:
AFAIK virtiofs does honor the use-after-unlink case gracefully. Maybe only do this for 9p ?

Reply:
This is a hack to get me moving. Essentially this shouldn't be needed, so I do not want to carry this on, regardless of the FS used.